obspy/core/trace.py

```diff
@@ -2597,9 +2598,10 @@ def remove_response(self, output="VEL", water_level=60, pre_filt=None,
             ax6 = fig.add_subplot(326, sharex=ax4)
             for ax_ in (ax1, ax2, ax3, ax4, ax5, ax6):
                 ax_.grid(zorder=-10)
-            ax1.text(0.05, 0.1, 'pre_filt: %s' % str(pre_filt),
-                     ha="left", va="bottom", transform=ax1.transAxes,
-                     fontsize="large", bbox=bbox, zorder=5)
+            text = 'pre_filt: ({:.3g} {:.3g} {:.3g} {:.3g})'.format(*pre_filt)
```

Comment:
Changing this line to 

```python
text = 'pre_filt: [{:.3g}, {:.3g}, {:.3g}, {:.3g}]'.format(*pre_filt)
```
should fix the tests and is IMHO also a bit more legible. Or you have to update the baseline test image.